index.js

```diff
@@ -1,3 +1,4 @@
+const { hasUncaughtExceptionCaptureCallback } = require('process');
```

Comment:
This is not being used, It may have gotten put here from accidently hitting `Tab` to auto complete to the wrong code. You can tell if it's not being used by looking at if it's highlighted or darkened. Feel free to take code out that's not being used.